Add content to playbooks (overview part) (#427)
SEKOIA-IO/documentation: _shared_content/automate/debug-playbooks.md

# Debug playbooks

## Rate limiting

> If too many actions are pushed, it may block running playbooks. In order to address this issue, playbooks have a rate limiting the action runs.

Reviewer: ```suggestion
If too many actions are pushed, it may block running playbooks. In order to address this issue, playbooks have rate limiting applied on their action runs.
```
